e2e/pages/app/partition-map/partition-map.testpage.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component } from '@angular/core';
+import { PopoverDirective } from '@ux-aspects/ux-aspects';
+
+@Component({
+    selector: 'app-partition-map',
+    templateUrl: './partition-map.testpage.component.html'
```

Comment:
I don't think this is documented anywhere, but we want new test components to use `OnPush` change detection, since we want to make sure all of our components work in that mode.